app/policies/application_policy.rb

```diff
@@ -7,31 +7,31 @@ def initialize(user, record)
   end
 
   def index?
-    false
+    user.is_a?(AdminUser)
```

Comment:
maybe we don't need these changes in `app/policies/application_policy.rb`

I'd have `app/policies/admin/application_policy.rb` define the policies for `AdminUser`

thoughts?

Reply:
I say it because `User` and `AdminUser` are different models, so I think policies should be different as well and would be easier to manage

Reply:
That'd be an improvement but Pundit reflects on policies using the model, so it uses the same policies as we normally use. To change this we would need to implement some way of getting policies from the `Admin` namespace if it's for Active Admin.

I completely agree with the comment but I'd add it as an enhancement issue.

Reply:
@santib I created the issue to deal with that, but I think this should be merged ASAP https://github.com/rootstrap/rails_api_base/issues/212

Reply:
I agree 💯